apps/meteor/client/views/admin/moderation/ModerationConsolePage.tsx

```diff
@@ -1,37 +1,52 @@
-import { useTranslation, useRouteParameter, useToastMessageDispatch } from '@rocket.chat/ui-contexts';
-import React from 'react';
+import { Tabs, TabsItem } from '@rocket.chat/fuselage';
+import { useTranslation, useRouteParameter, useToastMessageDispatch, type TranslationKey } from '@rocket.chat/ui-contexts';
+import React, { useCallback, useState } from 'react';
 
-import { Contextualbar } from '../../../components/Contextualbar';
 import Page from '../../../components/Page';
 import { getPermaLink } from '../../../lib/getPermaLink';
+import ModConsoleReportDetails from './ModConsoleReportDetails';
 import ModerationConsoleTable from './ModerationConsoleTable';
-import UserMessages from './UserMessages';
+import ModConsoleUsersTable from './UserReports/ModConsoleUsersTable';
+
+const tabs = ['Messages', 'Users'];
 
 const ModerationConsolePage = () => {
 	const t = useTranslation();
 	const context = useRouteParameter('context');
 	const id = useRouteParameter('id');
 	const dispatchToastMessage = useToastMessageDispatch();
+	const [tab, setTab] = useState('Messages');
+
+	const handleRedirect = useCallback(
+		async (mid: string) => {
+			try {
+				const permalink = await getPermaLink(mid);
+				window.open(permalink, '_self');
+			} catch (error) {
+				dispatchToastMessage({ type: 'error', message: error });
+			}
+		},
+		[dispatchToastMessage],
+	);
 
-	const handleRedirect = async (mid: string) => {
-		try {
-			const permalink = await getPermaLink(mid);
-			// open the permalink in same tab
-			window.open(permalink, '_self');
-		} catch (error) {
-			dispatchToastMessage({ type: 'error', message: error });
-		}
-	};
+	const handleTabClick = (tab: string) => () => setTab(tab);
 
 	return (
 		<Page flexDirection='row'>
 			<Page>
 				<Page.Header title={t('Moderation')} />
 				<Page.Content>
-					<ModerationConsoleTable />
+					<Tabs>
```

Comment:
This `Tabs` must live outside `Page.Content` to get this result: 

<img width="371" alt="Screenshot 2023-10-19 at 18 38 38" src="https://github.com/RocketChat/Rocket.Chat/assets/27704687/79349a1f-b101-43fd-b8f9-a31f847ef896">

Reply:
Thank you, done!